lua_examples/telnet/telnet.lua

```diff
@@ -125,15 +125,16 @@ local function telnet_listener(socket)
     node.input(line)
   end
 
-  local function disconnect(s)
+  local function discontect(s)
```

Comment:
Spelling

Reply:
Oups, I thought I corrected this one form the initial comit.